shacl12-test-suite/tests/core/node/uniqueMembers-001.ttl

```diff
@@ -0,0 +1,86 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix ex: <http://datashapes.org/sh/tests/core/node/uniqueMembers-001.test#> .
+@prefix mf: <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sht: <http://www.w3.org/ns/shacl-test#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ex:ListShape
+  rdf:type sh:NodeShape ;
+  sh:uniqueMembers true ;
+  # Satisfies all constraints
+  sh:targetNode ( 1 2 ), rdf:nil ;
+  # Violates uniqueMembers constraint
+  sh:targetNode _:b1, ex:notAList ;
+.
+
+_:b1
+  rdf:first 1 ;
+  rdf:rest ( 2 1 2 2 ) .
+
+<>
+  rdf:type mf:Manifest ;
+  mf:entries (
+      <uniqueMembers-001>
+    ) ;
+.
+
+<uniqueMembers-001>
+  rdf:type sht:Validate ;
+  rdfs:label "Test of sh:uniqueMembers on node shape 001" ;
+  mf:action [
+      sht:dataGraph <> ;
+      sht:shapesGraph <> ;
+    ] ;
+  mf:result [
+      rdf:type sh:ValidationReport ;
+      sh:conforms "false"^^xsd:boolean ;
+      sh:result [
+          rdf:type sh:ValidationResult ;
+          sh:focusNode _:b1 ;
+          sh:resultMessage "Value does not conform to Shape ex:ListShape. See details for more information." ;
```

Comment:
Wrong message - no ex:ListShape here.

Reply:
I think this is the right message `ex:ListShape` is the NodeShape which requires unique memebrs